wordpress-mobile/WordPress-Android
# [Reader] Add number of followed Sites and Tags to reader analytics events

Fixes #19836

This PR introduces a new analytics event to track the number of followed sites and tags, following the suggestions listed in the attached ticket.

Additionally, it fixes an issue with the tags update logic, which previously always triggered UI updates, and removes duplicate followed sites returned by the server that were causing unnecessary UI updates.

This is a work in progress.

To test:
- Would be easier to set a few breakpoints, instead of waiting for events to appear in Live View
- Open the Reader
- Go to Subscriptions. At the end of the network calls that refresh the reader, the analytics code should be called twice. One for tags and one for sites (*). Still working in progress
- Try with PTR. The analytics code should not be called, after the first time. 
- Start following a new Tag to the reader. The analytics code should be called after a while.

-

## PR Submission Checklist:


-----

## Testing Checklist:
